packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/list-projects-area/ProjectDownloadMenuItemExtension.tsx

```diff
@@ -16,18 +16,23 @@ import ListItemIcon from '@mui/material/ListItemIcon';
 import ListItemText from '@mui/material/ListItemText';
 import MenuItem from '@mui/material/MenuItem';
 import { useContext } from 'react';
+import { useProjectCapabilities } from '../../../hooks/useProjectCapabilities';
 import { ProjectContextMenuEntryProps } from './ProjectActionButton.types';
 
 export const ProjectDownloadMenuItemExtension = ({ project, onClose }: ProjectContextMenuEntryProps) => {
   const { httpOrigin } = useContext<ServerContextValue>(ServerContext);
 
+  const { data, loading } = useProjectCapabilities(project.id);
+  const canDownload: boolean = data?.viewer.project.capabilities.canDownload ?? false;
```

Comment:
This should be in the project object received as props

Reply:
Done